.github/workflows/deploy.yml

```diff
@@ -1,19 +1,21 @@
 name: Build & Deploy
 on:
   push:
-    branches: [main]
+    branches: [21-set-up-cicd-pipeline-with-do] #Needs to be changed to release when on main.
 jobs:
   deploy:
     runs-on: ubuntu-latest
     steps:
     - name: Deploy Node Monitor
```

Comment:
`name: Deploy Node Monitor` should be directly after `deploy:` right? I'm looking at Quint's `deploy.yaml` file right now.